OneSignalSDK/unittest/src/test/java/com/test/onesignal/GenerateNotificationRunner.java

```diff
@@ -2121,6 +2125,7 @@ public void notificationWillShowInForeground(OSNotificationGenerationJob.ExtNoti
    }
 
    @Test
+   @Config(shadows = { ShadowTimeoutHandler.class })
    public void testNotificationWillShowInForegroundHandler_workTimeLongerThanTimeout() throws Exception {
```

Comment:
This test is now taking ~35 seconds to complete when it wasn't before. 

I investigated this a bit and it is due to the `Thread.sleep` in `notificationWillShowInForeground` in the 
`com.test.onesignal.GenerateNotificationRunner$NotificationExtensionService_workTimeLongerThanTimeout` class never being interrupted.

The other app specific one `notificationWillShowInForeground` is being interrupted correctly however.

Reply:
there is no need for the 35 seconds, due that we are reducing the timeout time to 1, to avoid waiting that long. And made the thread sleep of 3 seconds